courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<p>I'd remove this section, since it's just repeating the learning objectives.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Just a nice little summary at the end to remind people of everything we just did without having to go back up to the top.</p>